blobber/amazons3_backend.py

```diff
@@ -55,6 +55,4 @@ def upload_to_AmazonS3(hashalgo, blobhash, data_file, headers, metadata):
             key.set_acl('public-read')
 
     # return the blob URL
-    amazon_baseurl = "http://" + BUCKET + ".s3.amazonaws.com"
-    blob_url = urlparse.urljoin(amazon_baseurl, _key)
-    return blob_url
+    return key.generate_url(expires_in=0)
```

Comment:
You should also remove urlparse from the top of the file as it's no longer needed. This is causing travis to fail:
https://travis-ci.org/mozilla/build-blobber/builds/257412292?utm_source=github_status&utm_medium=notification